pkg/providers/aws/provider_postgres.go

```diff
@@ -763,11 +763,11 @@ func (p *PostgresProvider) configureRDSVpc(ctx context.Context, rdsSvc rdsiface.
 
 func buildPostgresInfoMetricLabels(cr *v1alpha1.Postgres, instance *rds.DBInstance, clusterID, instanceName string) map[string]string {
 	labels := buildPostgresGenericMetricLabels(cr, clusterID, instanceName)
-	if instance != nil {
-		labels["status"] = *instance.DBInstanceStatus
+	if instance != nil && len(string(cr.Status.Phase)) != 0 {
+		labels["statusAWS"] = *instance.DBInstanceStatus
+		labels["statusPhase"] = string(cr.Status.Phase)
 		return labels
 	}
-	labels["status"] = "nil"
```

Comment:
Do we not want to keep this to report back that the instance is nil?

Reply:
already populated on https://github.com/integr8ly/cloud-resource-operator/pull/137/files#diff-8fd2a41e86b5db36e6443ed519133f75L765

Reply:
moving phase status to an new function and restoring this metric to original format.